packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -56,7 +56,7 @@ import { MediaStreamType } from '@azure/communication-calling';
 import { MicrosoftTeamsUserKind } from '@azure/communication-common';
 import type { NetworkDiagnosticChangedEventArgs } from '@azure/communication-calling';
 import { PartialTheme } from '@fluentui/react';
-import { PermissionConstraints } from '@azure/communication-calling';
+import { PermissionConstraints as PermissionConstraints_2 } from '@azure/communication-calling';
```

Comment:
I know we maybe have a precedent here to use a number but could we call it something else more specific to this dropdown? or callReadiness? the naming collision is with Rooms right?

Reply:
Rush build did this automatically, super weird...